### ISSUE TITLE
Single-quotted durations are not being parsed properly

### ISSUE THREAD

**Issue by antoiner77** [influxdata/telegraf#2023]
### Relevant telegraf.conf:
```
[[inputs.cloudwatch]]
  region = 'eu-west-1'
  period = '1m'
  delay = '1m'
  interval = '1m'
  namespace = 'AWS/ELB'
  [[inputs.cloudwatch.metrics]]
    names = ['RequestCount', 'HTTPCode_Backend_4XX', 'Latency']
    [[inputs.cloudwatch.metrics.dimensions]]
      name = 'LoadBalancerName'
      value = 'awseb-e-8-xxxxxxx'
```
### System info:
Telegraf v1.1.0 (git: release-1.1.0 8ecfe13bf813d5512267d90e7b3ad2f62aba5842)
AWS linux 
### Steps to reproduce:
1- Use my conf
2- run Telegraf
### Expected behavior:
Data import with no issue
### Actual behavior:
```
2016/11/09 16:11:30 E! ERROR in input [inputs.cloudwatch]: Errors encountered: [InvalidParameterValue: The parameter StartTime must not equal parameter EndTime.
	status code: 400, request id: cada001c-a68e-11e6-b82d-af8d232318ca], [InvalidParameterValue: The parameter StartTime must not equal parameter EndTime.
	status code: 400, request id: cada26a6-a68e-11e6-8558-d13147ee8794], [InvalidParameterValue: The parameter StartTime must not equal parameter EndTime.
	status code: 400, request id: cada4e6b-a68e-11e6-9bce-1ba48e9b8750], [InvalidParameterValue: The parameter StartTime must not equal parameter EndTime.
	status code: 400, request id: cadac311-a68e-11e6-9a00-b37caae06e6d], [InvalidParameterValue: The parameter StartTime must not equal parameter EndTime.
	status code: 400, request id: cada7544-a68e-11e6-91a2-6f4e21ef2c81], [InvalidParameterValue: The parameter StartTime must not equal parameter EndTime.
```
### Additional info:
Working great until i rebuild my stack :/

**Comment by sparrc** [comment]
> Working great until i rebuild my stack :/

what do you mean by this? was this working in a previous version?

**Comment by sparrc** [comment]
cc @johnrengelman

**Comment by antoiner77** [comment]
Same version i guess, i'm using the repo to install telegraf, my last test was a week ago but there as been no change to the Cloudwatch plugin for that period

**Comment by sparrc** [comment]
version 1.1 was released just a few days ago, you may have been using version 1.0.1 before.

**Comment by antoiner77** [comment]
Probably yes, but the "time related function" on the plugin haven't change :/. Strange behavior. I did not say but I of course use the same configuration file during my tests

**Comment by johnrengelman** [comment]
I don't see how this could happen unless you set `period` to 0 - https://github.com/influxdata/telegraf/blob/master/plugins/inputs/cloudwatch/cloudwatch.go#L351-L352

**Comment by sparrc** [comment]
might be due to toml parsing of the duration field, which did change in 1.1

@antoiner77 could you change your `'` to `"`?

**Comment by johnrengelman** [comment]
oh yeah...that's totally it. gah. stupid single quotes. they come back as 0 if you have something like `'1m'`

**Comment by antoiner77** [comment]
`wrote batch of 13 metrics`. Yea that was the issue. Thank you guys

**Comment by sparrc** [comment]
OK, I will get a fix in for this, 1.1.1 here we come, whatta day!! :sob:
